nudge/src/emailList.ts

```diff
@@ -1,29 +1,27 @@
 import { generateKeyMetadata } from './generateKeyMetadata'
 
 export async function extractEmails(results: any[]): Promise<Key[]> {
-  let keysData = results.map((result: any) =>
-    enrichKeyWithEmailAddress(result.lockAddress, result.keyId)
-  )
+  let keysData = results.map((result: any) => enrichKeyWithEmailAddress(result))
```

Comment:
```suggestion
  let keysData = results.map(enrichKeyWithEmailAddress)
```

Really minor suggestion, up to you if you want to use it

Reply:
It looks like ts isn't a huge fan of this. I will probably need to tighten up the types but we can do the tightening before we move this from staging to prod!